eks/vpn.tf

```diff
@@ -0,0 +1,46 @@
+resource "aws_eip" "vpn_eip" {
+  instance = aws_instance.vpn.id
+  vpc = true
+}
+
+resource "aws_instance" "vpn" {
+  ami = "ami-049aea444f70407b8"
+
+  instance_type = "t2.micro"
+
+  key_name = var.ec2_keypair
+  associate_public_ip_address = true
+  subnet_id = aws_subnet.public[0].id
+  vpc_security_group_ids = [
+    aws_security_group.node.id]
```

Comment:
This is a weird way to format this.

Use `terraform fmt` in the directory to format things correctly.